CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ## [unreleased] (Added 🚀 | Changed | Removed 🗑 | Fixed 🐞 | Chore 👨‍💻 👩‍💻)
 
+### Changed
+
+-   Metric descriptions are shown via tooltip in the legend [#3162](https://github.com/MaibornWolff/codecharta/pull/3162)
+    ![alt text](https://user-images.githubusercontent.com/27358421/205251507-100b1e1a-d86e-44f3-89cf-f32bf295aabd.png)
```

Comment:
If I compare this to other image tags in this changelog, I notice that there is a <br> missing a line before and other images do not use [alt text] but [image instead]

Reply:
In case you need some help, out contributing.md has an example on how to format images correctly https://github.com/MaibornWolff/codecharta/blob/main/CONTRIBUTING.md#how-should-you-format-a-changelog-entry

Reply:
I changed "alt text" to "image" now. 
I haven't seen where the line is missing